session.go

```diff
@@ -250,6 +254,9 @@ runLoop:
 		if err := s.sendPacket(); err != nil {
 			s.close(err)
 		}
+		if !s.receivedTooManyUndecrytablePackets.IsZero() && s.receivedTooManyUndecrytablePackets.Add(protocol.PublicResetTimeout).Before(time.Now()) {
```

Comment:
Add `now := time.Now()` just above this line and use that variable here and in the next 2 conditions.